src/main/java/seedu/application/logic/parser/ParserUtil.java

```diff
@@ -91,4 +88,19 @@ public static Status parseStatus(String status) throws ParseException {
         return new Status(trimmedStatus);
     }
 
+    /**
+     * Parses a {@code String Industry} into a {@code Industry}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code industry} is invalid.
+     */
+    public static Industry parseIndustry(String industry) throws ParseException {
+        requireNonNull(industry);
+        String trimmedIndustry = industry.trim();
+        if (!Industry.isValidIndustry(trimmedIndustry)) {
+            throw new ParseException(Industry.MESSAGE_CONSTRAINTS);
```

Comment:
I think line 101 is not covered by test. Maybe can take a look later when reviewing test coverage